packages/hardhat-verify/src/internal/tasks/sourcify.ts

```diff
@@ -0,0 +1,31 @@
+import chalk from "chalk";
+import { subtask } from "hardhat/config";
+
+import {
+  TASK_VERIFY_SOURCIFY,
+  TASK_VERIFY_SOURCIFY_DISABLED_WARNING,
+} from "../task-names";
+
+/**
+ * Main Sourcify verification subtask.
+ *
+ * Verifies a contract in Sourcify by coordinating various subtasks related
+ * to contract verification.
+ */
+subtask(TASK_VERIFY_SOURCIFY, async () => {
+  // code here
```

Comment:
Is this placeholder on purpose?

Reply:
It is. The Sourcify team will implement the task, this PR covers only the boilerplate to make sure it works seamlessly with Etherscan.